doc/testimonials/testimonials.rst

```diff
@@ -16,15 +16,8 @@ Who is using scikit-learn?
 
 .. raw:: html
 
-    <div class="logo">
-
-.. image:: images/jpmorgan.png
-    :width: 120pt
-    :target: https://www.jpmorgan.com
-
-.. raw:: html
-
-    </div>
+   <div class="sk-sponsor-div">
+   <div class="sk-sponsor-div-box">
```

Comment:
I find that re-using the `sk-sponsor-div` and `sk-sponsor-box` is semantically misleading. Maybe we should better introduce a new CSS classes such as `sk-testimonial-gallery` and  `sk-testimonial-box`.